testslide/__init__.py

```diff
@@ -181,29 +196,31 @@ class AggregatedExceptions(Exception):
     Aggregate example execution exceptions.
     """
 
-    def __init__(self):
+    def __init__(self) -> None:
         super(AggregatedExceptions, self).__init__()
-        self.exceptions = []
+        self.exceptions: List[Union[Exception, BaseException]] = []
```

Comment:
`List[BaseException]` shouldn't this work? Exceptions are BaseExceptions after all...